src/NerdMinerV2.ino.cpp

```diff
@@ -32,14 +32,25 @@ TFT_eSprite background = TFT_eSprite(&tft);  // Invoke library sprite
 
 unsigned long start = millis();
 const char* ntpServer = "pool.ntp.org";
+byte brightness = 255;
```

Comment:
I'd suggest to use unsigned int, if you anyway need to cast it.

Reply:
But the byte is smaller and the max size of brightness is 255, in memory is more efficient.